src/model/constants.rs

```diff
@@ -1,6 +1,6 @@
 // Model constants
-pub const MULTIPLIER: f64 = 60.0;
-pub const DEFAULT_VOLATILITY: f64 = 5.0 * MULTIPLIER;
+pub const MULTIPLIER: f64 = 66.67;
```

Comment:
Would probably leave these constants untouched for now until we finalize an approach for adjusting the resulting distribution

Reply:
ok